pom.xml

```diff
@@ -14,7 +14,7 @@
     <description>AYS Services</description>
     <groupId>com.ays</groupId>
     <artifactId>ays-be</artifactId>
-    <version>v1.0.0</version>
+    <version>1.0.0</version>
```

Comment:
may i ask you why you've removed the 'v'?

Reply:
I reviewed the other dependencies and noticed that none of them have the `v` prefix in the version section. Therefore, I made the correction here. All dependencies are defined in the format `<version>1.0.0<version/>`.